test/system/252-quadlet.bats

```diff
@@ -150,14 +150,13 @@ EOF
     run_quadlet "$quadlet_file"
     service_setup $QUADLET_SERVICE_NAME
 
+    # check that we can read the logs from the container with podman logs
```

Comment:
This comment was appropriate when this code ran below journalctl, but is less so now. A more helpful comment now would be something like "quadlet container emits a standard STARTED message once it's running". This is not worth re-pushing for.

Reply:
Oh, good catch! I failed to notice that the original assertion was `==`. If that's how quadlet works, and it guarantees emitting `STARTED CONTAINER` and absolutely no other output, then it's important to keep the check that way.

Reply:
I will update the PR accordingly but notice that relying on `passthrough` being the default log-driver may be risky as it can be changed in the future.  An explicit test for it may be needed.

Reply:
PTAL at the new version, should address everything.  I also hardcoded the `passthrough` driver to make sure the test continues as intended.